flags.go

```diff
@@ -19,14 +20,17 @@ const statsdAddress = "127.0.0.1:8200"
 
 var stats *statsd.Client
 
+var staleness_threshold time.Duration = 5 * time.Minute
+var staleMtx = sync.RWMutex{}
```

Comment:
I think we can just use one big mutex for all globals. The more mutexes we have, the harder it gets to reason about what's going on.

Reply:
Changed my mind, this one is ok cuz it's written to at very different times.